karmada-CLI-installtion-and-deploy-example/finish.md

```diff
@@ -0,0 +1,3 @@
+**Summary**
+
+In this scenario, we learned how to initialize the Karmada control plane, join a cluster.
```

Comment:
In this scenario, we learned how to initialize the Karmada control plane and join a cluster.

Reply:
yeah, thank you for your pointing out, that's a typo.
I think fix done.